docs/content/developer/getting-started/independent-ticketing-system/package.mdx

```diff
@@ -0,0 +1,183 @@
+    NFTContract ->> NFTContract: Validate sender is NFT owner
+    NFTContract ->> NFTContract: Validate recipient is whitelisted
+    NFTContract ->> Blockchain: Update price and initiate resale
+
+    User ->> NFTContract: buy_resale(coin, initiated_resale, ctx)
+    NFTContract ->> NFTContract: Validate sender is buyer
+    NFTContract ->> Blockchain: Split coin for royalty and seller payment
+    NFTContract ->> Blockchain: Transfer royalty fee to creator
+    NFTContract ->> Blockchain: Transfer payment to seller
+    NFTContract ->> Blockchain: Public transfer of TicketNFT to buyer
+    NFTContract ->> NFTContract: Delete InitiateResale object
+
+    User ->> NFTContract: burn(nft, ctx)
+    NFTContract ->> NFTContract: Validate sender is NFT owner
+    NFTContract ->> Blockchain: Delete NFT object
+
+    User ->> NFTContract: whitelist_buyer(user, nft)
+    NFTContract ->> NFTContract: Add user to NFT whitelist
+
+    User ->> NFTContract: create_TicketNFT(name, creator, owner, ... , ctx)
+    NFTContract ->> NFTContract: Create a new TicketNFT (Test purpose)
+
+    User ->> NFTContract: create_TotalSeat(value, ctx)
+    NFTContract ->> NFTContract: Create TotalSeat object (Test purpose)
+```
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en) >= v20.18.0
+- [npx](https://www.npmjs.com/package/npx) >= 10.8.2
+- [iota](https://github.com/iotaledger/iota/releases) >= 0.7.3
+
+## [Create a Move Package](../create-a-package.mdx)
+
+Run the following command to create a Move package:
+
+```bash
+iota move new independent_ticketing_system
+```
+
+## Package Overview
+
+### [`init`](https://github.com/iota-community/independent-ticketing-system/blob/6c6cd27bbf402dc3e1a219a2d6e5556eb968bb59/independent_ticketing_system/sources/independent_ticketing_system.move#L71-L85)
+
+In MOVE, the `init` function only runs once at the time of package publication. Read more: [`init`](../../iota-101/move-overview/init)
+The `init` function will create three [`shared_object`](../../iota-101/objects/object-ownership/shared.mdx) type objects: 
+- [`Creator`](https://github.com/iota-community/independent-ticketing-system/blob/6c6cd27bbf402dc3e1a219a2d6e5556eb968bb59/independent_ticketing_system/sources/independent_ticketing_system.move#L20-L23) will store the address of the package creator.
+- [`TotalSeat`](https://github.com/iota-community/independent-ticketing-system/blob/6c6cd27bbf402dc3e1a219a2d6e5556eb968bb59/independent_ticketing_system/sources/independent_ticketing_system.move#L25-L28) - it will contain the total number of seat for the event which will update as per minting the tickets.
+- [`AvailableTicketsToBuy`](https://github.com/iota-community/independent-ticketing-system/blob/6c6cd27bbf402dc3e1a219a2d6e5556eb968bb59/independent_ticketing_system/sources/independent_ticketing_system.move#L38-L41) - it will contains the array of all the nft which are minted by the  creator and are available to buy. To buy tickets first user needs to get whitelisted. 
```

Comment:
```suggestion
- [`AvailableTicketsToBuy`](https://github.com/iota-community/independent-ticketing-system/blob/6c6cd27bbf402dc3e1a219a2d6e5556eb968bb59/independent_ticketing_system/sources/independent_ticketing_system.move#L38-L41) - it will contain the array of all the NFT which are minted by the creator and are available to buy. To buy tickets, the first user needs to be whitelisted. 
```